src/domain/app/AppConfig.ts

```diff
@@ -74,21 +74,18 @@ class AppConfig {
   }
 
   static get matomoConfiguration() {
-    const matomoUrlBase = '//webanalytics.digiaiiris.com/js/';
+    const matomoUrlBase = process.env.NEXT_PUBLIC_MATOMO_URL_BASE;
     const matomoEnabled = process.env.NEXT_PUBLIC_MATOMO_ENABLED;
     const matomoSiteId = process.env.NEXT_PUBLIC_MATOMO_SITE_ID;
     const getMatomoUrlPath = (path: string) => `${matomoUrlBase}${path}`;
```

Comment:
Should the environment variable be used instead? If not, then the env-variables related to this should be removed.